cfme/tests/configure/test_tag.py

```diff
@@ -124,6 +126,29 @@ def test_map_tagging_crud(appliance, category, soft_assert):
                                           .format(map_tag_entity.label))
 
 
+@pytest.fixture
+def csv_tag_file(create_vm, category, tag):
+    temp_file = tempfile.NamedTemporaryFile(suffix=".csv")
+    csv_data = f'name,category,entry\n{create_vm.name},{category.display_name},{tag.display_name}'
```

Comment:
Shouldn't you be using a random alphanumeric for the tag, instead of trying to import a tag that matches an existing one?

The `tag` fixture that you're using here is a random tag that has already been created on the category.

Reply:
yes. The import is to add the existing tag value to a VM. If the tag or category or VM do not exist, the upload fails. the terms here are confusing between a category, entry, tag etc. . I believe i have the bahavior ad it is in the BZ.

Reply:
Can you add to the docblock for the test case the description that you're importing tag mappings to a VM, and not just tag definitions.

Reply:
Done